spark-common/src/main/java/me/lucko/spark/common/SparkPlatform.java

```diff
@@ -142,7 +142,7 @@ public SparkPlatform(SparkPlugin plugin) {
 
         this.tickHook = plugin.createTickHook();
         this.tickReporter = plugin.createTickReporter();
-        this.tickStatistics = this.tickHook != null ? new TickStatistics() : null;
+        this.tickStatistics = this.tickHook != null || this.tickReporter != null ? new TickStatistics() : null;
```

Comment:
The assumption in spark currently is that if there is a `tickReporter` then there is also a `tickHook`.  I think Minestom should be able to inplement both with the events that are available?